### PR TITLE
Update “Join a team” styles.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review.

### CODE REVIEW

File: shared/teams/join-team/index.native.js [hunk 1]
```diff
@@ -1,52 +1,77 @@
...
     <Box style={globalStyles.flexBoxColumn}>
-      <Box
-        style={{
-          ...globalStyles.flexBoxColumn,
-          alignItems: 'center',
-          justifyContent: 'center',
-          marginTop: globalMargins.small,
-        }}
-      >
+      <Box style={styles.container}>
```
Comment (chrisnojima) [inline shared/teams/join-team/index.native.js]: likely better to use box2 with gapStart/End = true and fullWidth

**Comment by adamjspooner** [comment]
@chrisnojima, this is ready for another review.

File: shared/teams/join-team/index.native.js [hunk 2]
```diff
@@ -1,52 +1,80 @@
...
+  info: {
+    maxWidth: 280,
+    textAlign: 'center',
+  },
+  input: {
+    borderRadius: 4,
+    borderStyle: 'solid',
+    borderWidth: 1,
+    marginTop: globalMargins.small,
+    paddingBottom: globalMargins.small,
```
Comment (chrisnojima) [inline shared/teams/join-team/index.native.js]: if they're all the same you can just use padding

Comment (adamjspooner) [reply shared/teams/join-team/index.native.js]: I thought it should, but the top and bottom paddings are ignored.

Comment (chrisnojima) [reply shared/teams/join-team/index.native.js]: it shoudn't maybe something else is happening. what does react-devtools say. maybe you need flexShrink: 0

Comment (adamjspooner) [reply shared/teams/join-team/index.native.js]: This is why: https://github.com/keybase/client/blob/master/shared/common-adapters/input.native.js#L255-L256

Comment (chrisnojima) [reply shared/teams/join-team/index.native.js]: you should use PlainInput if you're going to change the styling so much

Comment (adamjspooner) [reply shared/teams/join-team/index.native.js]: Cool beans. Will do.

**Comment by adamjspooner** [comment]
@chrisnojima, this is ready for another review.

**Pull request merged**